components/odh-notebook-controller/controllers/suite_test.go

```diff
@@ -79,6 +88,37 @@ const (
 	odhNotebookControllerTestNamespace = "redhat-ods-applications"
 )
 
+// getOAuthProxyImageFromParams reads the oauth-proxy-image value from config/base/params.env
+// Returns empty string on error to allow graceful handling by caller
+func getOAuthProxyImageFromParams() string {
+	paramsFile := filepath.Join("..", "config", "base", "params.env")
```

Comment:
what about using Kustomize, adding env variable for the controller deployment that uses variable from `oauth-proxy-image=` value in params.env?